env.sh

```diff
@@ -2,14 +2,15 @@
 
 export MACOSX_DEPLOYMENT_TARGET=10.10
 
-export BRAVE_TOR_VERSION="0"
-export TOR_VERSION="0.3.5.11"
+export BRAVE_TOR_VERSION=""
```

Comment:
We use the `BRAVE_TOR_VERSION` to keep track of new releases for the same `tor version` if there are any issues with libraries/platform dependencies. Can you please revert it to 0, thanks!

Reply:
Fixed